atd-etl/cris_import/utils/process_pdfs.py

```diff
@@ -19,42 +19,56 @@
 logger = get_logger()
 
 
-def is_new_cr3_form(page):
-    """Determine if the CR3 is following the older or newer format.
-
+def get_cr3_version(page, page_width):
+    """Determine the CR3 from version.
+    
     The check is conducted by sampling if various pixels are black.
 
     Args:
         page (PIL image): the pdf page as an image
+        page_width (int): the width of the PDF in points
 
     Returns:
-        bool: true if the provided page passes the sampling tests
+        str: 'v1', 'v2_large', or 'v2_small'
     """
-    new_cr3_form = True
+    if page_width < 700:
+        # the most recent version is the same layout as v2-large
+        # but it is delivered in a smaller page size
+        # this took effect 27 Aug 2024
+        return "v2_small"
```

Comment:
bleh—i need to rework this now that we know that all PDFs from all time will be delivered in this smaller format.
https://austininnovation.slack.com/archives/CM9MK950S/p1724764680059889?thread_ts=1724758134.799859&cid=CM9MK950S